### ISSUE TITLE
How to plot a weight histogram 

### ISSUE THREAD

**Issue by Swoorup** [scalanlp/breeze#837]
How do you plot a weight histogram from `stats.hist` to `plot.hist`? 

It doesn't seem possible?

**Comment by dlwh** [comment]
i don't think it is trivial. `stats.hist` was written after me wanting to maintain the plotting library. You should. pass the data directly to plot.hist and it'll do its own thing
